syntax = "proto2";

package proto.db;

message nop_command {
}